examples/Button.stories.svelte

```diff
@@ -19,7 +20,7 @@
     component: Button,
     tags: ['autodocs'],
     args: {
-      children: 'Click me',
+      children: createRawSnippet(() => ({ render: () => 'Click me' })),
```

Comment:
I don't think this is a good example we want to document, and I don't think it's needed because of line 42 below?

If it's to satisfy types, then I think we should use type modifications instead like `as`.

Reply:
Changing to:

```diff
- children: createRawSnippet(() => ({ render: () => 'Click me' })),
+ children: 'Click me' as Snippet,
```

will produce the error:

```txt
Conversion of type 'string' to type 'Snippet<[]>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first. [2352]
```

So, the workaround is:

```diff
- children: 'Click me' as Snippet,
+ children: 'Click me' as unknown as Snippet,
```

Which is cumbersome. Shall we proceed with this?

Reply:
I provided another alternative, which can be reverted: 104d5e0cb9dc818c104834f31755e44eb59b4f41

Reply:
Wait but do you want a snippet or a string? Because those are not interchangeable

Reply:
Well, we want a string here, because we convert it to a snippet down in the template below:

https://github.com/storybookjs/addon-svelte-csf/pull/219/files/6b646f1b2e9a44c8e52814b23ae2d71f93797b07#diff-27706a42acb167034a3d3aa2f621b4a43e00b1bd177c0e87c2028fad8e14b398R41-R43

This procedure ensures you can easily modify the arg as a string in the Control panel, while it still being passed as a snippet to the component.
But it's slightly cumbersome, and I haven't found a way yet to support this pretty common use case better yet - ideas welcome!

Reply:
@xeho91 your alternative looks cleaner, but now it doesn't allow you to modify it in Controls anymore though. but let's stick with it. 👍